tube-player/modules/10-Add-media-player/README.md

```diff
@@ -137,7 +148,9 @@ Run the app to see the media player playing the video from YouTube. Seek, volume
 You will notice how the video stops before navigating back to the search results (observe the play button right before the navigation happens), but at the same time, we still want to update the navigation bar image (currently a random picture), the app icon, as well as the splashscreen (which are currently set to the template defaults).
 
 > [!NOTE]
-> In order to be able to play the media in WASM, because of CORS issue, you will need to either create a server project (e.g. using [YARP](https://chat.openai.com/share/d633a7b6-1b3c-4730-a60b-96dfd97baa0a)) or by using a public proxy (e.g. using [CORS Anywhere](https://github.com/Rob--W/cors-anywhere)).
+> In order to be able to play the media in WASM, because of a CORS issue, you will need to either create a server project (e.g. using [YARP](https://chat.openai.com/share/d633a7b6-1b3c-4730-a60b-96dfd97baa0a)) or by using a public proxy (e.g. using [CORS Anywhere](https://github.com/Rob--W/cors-anywhere)).
+
+For more information on `MediaPlayerElement` support, refer to the [MediaPlayerElement documentation](https://platform.uno/docs/articles/controls/MediaPlayerElement.html).
```

Comment:
```suggestion
> In order to be able to play the media in WASM, because of a CORS issue, you will need to either create a server project (e.g. using [YARP](https://chat.openai.com/share/d633a7b6-1b3c-4730-a60b-96dfd97baa0a)) or by using a public proxy (e.g. using [CORS Anywhere](https://github.com/Rob--W/cors-anywhere)).

For more information on `MediaPlayerElement` support, refer to the [MediaPlayerElement documentation](https://aka.platform.uno/mediaplayerelement).
```
Use the aka link instead, please ;)